templates/cicd/pipeline.yml

```diff
@@ -21,6 +21,8 @@ source:
 stages:{{range .Stages}}
     - # The name of the environment to deploy to.
       name: {{.Name}}
+      # Optional: flag for manual approval action before deployment.
+      requires_approval: {{.RequiresApproval}}
```

Comment:
What do you think of having this commented out, by default (like the test commands)? It will be false by default. and folks can uncomment it if they want it!

Reply:
Sounds nice! I addressed.